### PR TITLE
Adding library H3LIS200DL

### PR DESCRIPTION
**Pull request by jposada202020**

### CODE REVIEW

File: circuitpython_community_library_list.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Here is a listing of current CircuitPython Community Libraries. These libraries
 * [CircuitPython_DRV8830](https://github.com/CedarGroveStudios/CircuitPython_DRV8830.git) A driver for the DRV8830 DC motor controller.
 * [CircuitPython GC9A01](https://github.com/tylercrumpton/CircuitPython_GC9A01.git) Displayio driver for GC9A01 TFT LCD displays.
 * [CircuitPython gpio_expander](https://github.com/gpongelli/CircuitPython_gpio_expander.git) I2C GPIO expander support for PCA9534, PCA9535, PCA9555, TCA9534, TCA9535 and TCA9555 chips. ([PyPi](https://pypi.org/project/circuitpython-gpio-expander/)) \([Docs](https://github.com/gpongelli/CircuitPython_gpio_expander/blob/main/README.rst))
+* [CircuitPython BMP581](https://github.com/jposada202020/CircuitPython_H3LIS200DL.git) Driver for the ST H3LIS200DL Accelerometer sensor ([PyPi](https://pypi.org/project/circuitpython-h3lis200dl/)) \([Docs](https://circuitpython-h3lis200dl.readthedocs.io/en/latest/))
```
Comment (tekktrik) [inline circuitpython_community_library_list.md:23]: Looks like your library name doesn't match the name listed in the git repo URL, is that intentional?

**Review by tekktrik — CHANGES_REQUESTED** [review]

File: circuitpython_community_library_list.md (hunk 1 above)
Comment (jposada202020) [reply circuitpython_community_library_list.md:23]: Thank you!!!!!!!!! : No it isnot, a copy/paste mistake, ill fix it tonight :)

Comment (jposada202020) [reply circuitpython_community_library_list.md]: pls see https://github.com/adafruit/CircuitPython_Community_Bundle/pull/176/commits/035f2c3774d346b50c36fab3c89c1196aae9471a. Thanks

**Review by tekktrik — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**